galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -371,3 +371,9 @@ messages:
     args:
       - name: annotation
         type: string
+
+  - name: "GatewayDuplicateCertificate"
+    code: IST0137
+    level: Error
```

Comment:
I think this should be a warning. The config is 100% valid, it just doesn't behave as one might expect *in some cases* (due to https://github.com/envoyproxy/envoy/issues/6767).

Also can you link to https://github.com/envoyproxy/envoy/issues/6767 here, so we know to remove this analyzer if/when that issue is resolved?.

I would also change the warning to something like `"Duplicate certificate in multiple gateway, which may cause 404s if clients re-use HTTP2 connections"`. And hopefully the IST0137 doc will have more info about this (like the current istio.io doc)?

Reply:
sure will do that, refer to the issue as a comment in the code?

Reply:
Yeah I don't think we want it in the user facing info, just a comment in the code

Reply:
this done, will add documentation in the istio.io repository after this merge